src/api/app/helpers/webui/staging/workflow_helper.rb

```diff
@@ -69,7 +69,9 @@ def create_request_links(request, users_hash, groups_hash)
     css = 'review' if request[:missing_reviews].present?
     css = 'obsolete' if request[:state].in?(BsRequest::OBSOLETE_STATES)
     css += ' delete' if request[:request_type] == 'delete'
-    link_content = [request[:package]]
+    # special-case SLFO maintenance patchinfos for which the numbers
+    # in the package name are considered an implementation detail
```

Comment:
I don't think it adds value to comment this in the code, you already explained everything about this in the commit message introducing this code.